crates/wasm-mutate/src/lib.rs

```diff
@@ -316,8 +321,14 @@ impl<'wasm> WasmMutate<'wasm> {
         // subslice of data with a random slice of other data.
         //
         // First up start/end indices are picked.
-        let a = self.rng().gen_range(0..=data.len());
-        let b = self.rng().gen_range(0..=data.len());
+        let (a, b) = if data.is_empty() {
```

Comment:
For data here specifically it seems surprising that this would panic because `0..=0` is a range with one element to choose from so it seems that `gen_range` shouldn't panic.

Reply:
Yeah this is my b, I didn't think about how this is an inclusive range.